rust/kernel/print.rs

```diff
@@ -12,7 +12,12 @@ use core::fmt;
 use crate::bindings;
 use crate::c_types::{c_char, c_void};
 
-// Called from `vsprintf` with format specifier `%pA`.
+/// Called from `vsprintf` with format specifier `%pA`.
+///
+/// # Safety
+///
+/// The region between `buf` and `end` must be valid for writes. `buf` must be
+/// properly aligned. `ptr` must point to a valid instance of `fmt::Arguments`.
```

Comment:
What is "properly" aligned? Doesn't it require just a 1 byte alignment?

Reply:
Oh, right, those are C chars, sorry